mmv1/third_party/terraform/tests/resource_cloud_identity_group_test.go.erb

```diff
@@ -162,3 +162,60 @@ func testAccCheckCloudIdentityGroupDestroyProducer(t *testing.T) func(s *terrafo
 		return nil
 	}
 }
+
+func TestAccCloudIdentityGroup_cloudIdentityGroupsDynamicExample(t *testing.T) {
+	t.Parallel()
+
+
+	context := map[string]interface{}{
+		"org_domain":    getTestOrgDomainFromEnv(t),
```

Comment:
Due to a bit of unlucky timing, we're gonna have to capitalize most of the functions here, due to https://github.com/GoogleCloudPlatform/magic-modules/pull/7403 Rebasing your MM & provider repo will trigger the right changes locally.

Error:

```
# github.com/hashicorp/terraform-provider-google-beta/google-beta [github.com/hashicorp/terraform-provider-google-beta/google-beta.test]
google-beta/resource_cloud_identity_group_test.go:167:20: undefined: getTestOrgDomainFromEnv
google-beta/resource_cloud_identity_group_test.go:168:20: undefined: getTestCustIdFromEnv
google-beta/resource_cloud_identity_group_test.go:169:20: undefined: randString
google-beta/resource_cloud_identity_group_test.go:172:2: undefined: vcrTest
google-beta/resource_cloud_identity_group_test.go:174:17: undefined: testAccProviders
?   	github.com/hashicorp/terraform-provider-google-beta	[no test files]
FAIL	github.com/hashicorp/terraform-provider-google-beta/google-beta [build failed]
?   	github.com/hashicorp/terraform-provider-google-beta/version	[no test files]
FAIL
make: *** [test] Error 2
```

Reply:
@rileykarson 👋 It's been awhile, but we're back at this. 🙂 

I've rebased and regenerated the provider, but I'm still running into these same errors (at least locally).

```

Reply:
Those functions were permanently moved during that earlier refactoring, so you'll need to update your test code to refer to their new locations is all!

 Their interfaces are still identical so the changes are all pretty mechanical- along the lines of `getTestOrgDomainFromEnv ` -> `envvar.GetTestOrgDomainFromEnv`- see https://github.com/mdtro/magic-modules/blob/5fbc7fc8028aa3443819d8bbc2bd780bf2c74bd2/mmv1/third_party/terraform/services/cloudidentity/resource_cloud_identity_group_test.go.erb#L93-L116 for another definition within the file.

I think getTestOrgDomainFromEnv/getTestCustIdFromEnv are now in the `envvar` package and the rest in `acctest`, but cross-referencing another sample will confirm that.

Reply:
That makes a lot of sense. 😅 I should have read the documentation more closely. Thank you!

I think that sorted it out. 👍